### DISCUSSION TITLE
TradingView Secret Key

### DISCUSSION THREAD

**Discussion by ParityError** [FabCx/TradingView-Webhook-Bot#50] [Q&A]
Where do I find the secret key in **TradingView** to apply to `config.py`?

**Comment by fabston** [comment]
Hi,

TV does not provide you with a secret key. You’ll have to set the key yourself in `config.py` and reference that in the TV Alerts dialog. 

Hope that answers your question.

**Reply by ParityError** [reply]
Ah, that's where the confusion was. Thank you!

**Reply by fabston** [reply]
Your welcome :)
